ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh

```diff
@@ -67,7 +70,28 @@ then
   . network.sh
   . utils.sh
 
-  ssh -o 'ConnectTimeout=5' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -o 'ServerAliveInterval=90' core@\$BOOTSTRAP_PROVISIONING_IP TAR_FILE=/tmp/log-bundle-bootstrap.tar.gz sudo -E /usr/local/bin/installer-gather.sh --id bootstrap &&
+  # Pass master IPs to installer-gather script to collect info from nodes which didn't join the cluster
+  NODE_IPS=()
+  for (( n=0; n<\$NUM_MASTERS; n++ ))
+  do
+     node_name=\$(printf \$MASTER_HOSTNAME_FORMAT \$n)
+     node_ip=\$(sudo virsh net-dumpxml \$BAREMETAL_NETWORK_NAME | xmllint --xpath "string(//dns[*]/host/hostname[. = '\${node_name}']/../@ip)" -)
+     NODE_IPS+=("\$node_ip")
+  done
+  for (( n=0; n<\$NUM_WORKERS; n++ ))
+  do
+     node_name=\$(printf \$WORKER_HOSTNAME_FORMAT \$n)
+     node_ip=\$(sudo virsh net-dumpxml \$BAREMETAL_NETWORK_NAME | xmllint --xpath "string(//dns[*]/host/hostname[. = '\$node_name']/../@ip)" -)
+     NODE_IPS+=("\$node_ip")
+  done
+  for (( n=0; n<\$NUM_EXTRA_WORKERS; n++ ))
+  do
+     node_name=\$(printf \$WORKER_HOSTNAME_FORMAT \$n)
```

Comment:
It seems extra workers have another format that includes ``extraworker`` in the name. Current code seems to repeat logs extraction from the "regular" workers

Reply:
>It seems extra workers have another format that includes extraworker in the name

I can't seem to find references to this in the code :/

Reply:
I think it's this part: https://github.com/metal3-io/metal3-dev-env/blob/2b22c36350b047837bc466388f5a597d11886a20/vm-setup/roles/libvirt/templates/network.xml.j2#L42
No value passed for the hostname format of extra-workers by dev-scripts, so by default it takes the flavor name ``extraworker`` and appends the number

Reply:
Ah, that explains it, updated the script.